src/styles/components/_our-goal-box.scss

```diff
@@ -0,0 +1,46 @@
+@import '../abstracts/_abstracts-dir';
+
+.our-goal-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px $silver-chalice;
+  background-color: transparent;
```

Comment:
Since `transparent` is the default value of `background-color`, Do we really need to override this?

Reply:
No need krub, my bad.